integration-libs/epd-visualization/src/components/visual-picking/visual-picking-product-filter/visual-picking-product-filter.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="form-group search-wrapper">
+  <input
+    #queryInput
+    type="text"
+    [(ngModel)]="filter"
+    class="form-control"
+    placeholder="{{
+      'visualPicking.visualPickingProductFilter.input.placeholder' | cxTranslate
+    }}"
+  />
+
+  <cx-icon
+    [type]="iconTypes.SEARCH"
+    [attr.aria-label]="
+      'visualPicking.visualPickingProductFilter.searchButton.label'
+        | cxTranslate
+    "
+    class="search"
+    [hidden]="filter.length > 0"
+  ></cx-icon>
+
+  <cx-icon
+    [type]="iconTypes.RESET"
+    [attr.aria-label]="
+      'visualPicking.visualPickingProductFilter.resetButton.label' | cxTranslate
+    "
+    (mousedown)="filter = ''"
+    (keydown.enter)="filter = ''"
```

Comment:
here we use mousedown and keydown.enter event to clear filter. How about in mobile device? Can these two events still be triggered when user taps in mobile?

Reply:
I have been using the Chrome devtools device toolbar for simulation touch events (and I have used this for testing touch gestures for the animation slider). A tap does seem to get converted into mouse events, so the mousedown handle has been getting invoked for a touch tap in this simulated mode.